Sources/Apodini/Components/Component.swift

```diff
@@ -41,3 +41,10 @@ extension Component {
         }
     }
 }
+
+extension EventLoopFuture: Encodable where Value: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        let encodable = try self.wait()
```

Comment:
this is a blocking operation, right?

can we safely block the thread, or would that impact other parts of the server (ie not being able to process other incoming requests)?

Reply:
Yeah its blocking. I need to come up with another function that somehow returns an `EventLoopFuture`

Reply:
just a guess, but if we configure vapor in a way that each request is guaranteed to get its own thread, wouldn't we get away with blocking here?